parent/pom.xml

```diff
@@ -73,6 +73,7 @@
     <!-- Upstream -->
     <version.image.java.upstream.docker>1.5</version.image.java.upstream.docker>
     <version.image.java.upstream.s2i>2.3</version.image.java.upstream.s2i>
+    <version.image.webapp.upstream.s2i>latest</version.image.webapp.upstream.s2i>
```

Comment:
we shouldn't use latest in our deps if not for a very good reason. Can't we stick to a version, like for the other images ?

Reply:
not sure, there doesn't seem to be any other tag here: https://hub.docker.com/r/openshift/wildfly-101-centos7/tags/